client/components/cards/attachments.jade

```diff
@@ -30,8 +30,14 @@ template(name="attachmentDeletePopup")
   p {{_ "attachment-delete-pop"}}
   button.js-confirm.negate.full(type="submit") {{_ 'delete'}}
 
-template(name="attachmentsGalery")
-  .attachments-galery
+template(name="attachmentsGallery")
+  #viewer-overlay.hidden
+    #viewer-container
+      object#pdf-viewer(type="application/pdf")
+        span.pdf-preview-error Your device does not support previewing pdfs. Try downloading instead.
```

Comment:
I have not yet figured out how to add a new translation key for this. Any help is appreciated.

Reply:
Added at https://github.com/wekan/wekan/commit/83ae1d4a19925363d8f176553fb10a98a77420be